docroot/profiles/custom/cgov_site/modules/custom/cgov_yaml_content/src/Service/CgovYamlContentEventSubscriber.php

```diff
@@ -485,6 +516,7 @@ public function addSpanishTranslations(EntityPostSaveEvent $event) {
     // need to set it the same as English for it to work because it
     // is a special field and does not work the same way as others.
     if (!array_key_exists('moderation_state', $translatedFields)) {
+      /* @phpstan-ignore-next-line */
       $spanishTranslation->{'moderation_state'} = $entity->{'moderation_state'};
```

Comment:
Why do you need the ignore-next-line here? Is there another way of doing this.

Reply:
Resolved.